.github/workflows/build-ci.yml

```diff
@@ -15,26 +15,18 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v5
 
-      - name: Setup JDK 11
-        id: setup-java-11
+      - name: Setup JDKs
         uses: actions/setup-java@v5
         with:
           distribution: 'temurin'
-          java-version: '11'
-
-      - name: Setup JDK 17
-        id: setup-java-17
-        uses: actions/setup-java@v5
-        with:
-          distribution: 'temurin'
-          java-version: '17'
+          java-version: |
+            11
+            17
+            21
```

Comment:
Why are we always specifying 3 versions? Seems like we're often replacing just 11, or 11+17, or 11+21.